config.rb

```diff
@@ -55,7 +59,10 @@
 
 helpers do
   def include(path)
-    IO.read(path).sub(/^#[ \w].+$/, '')
+    if path.include? ".rb"
+      print("[debug] " + path)
+      IO.read(path).sub(/^#[ \w].+$/, '')
+    end
```

Comment:
Can you the explain the intent of this change, in particular why have you made the include check for the `.rb` extension? It's stripping out the `h1` heading of the files being included (which match `^#[ \w].+$`).

It's seems to only be called for markdown files, and break some of the documentation pages:

```shell
$ git grep include
source/documentation/advanced-tutorial.html.md.erb:<%= include "api-blueprint/Advanced Tutorial.md" %>
source/documentation/glossary.html.md.erb:<%= include "api-blueprint/Glossary of Terms.md" %>
source/documentation/mson/specification.html.md.erb:<%= include "mson/MSON Specification.md" %>
source/documentation/mson/tutorial.html.md.erb:<%= include "mson/Tutorial.md" %>
source/documentation/specification.html.md.erb:<%= include "api-blueprint/API Blueprint Specification.md" %>
source/documentation/tutorial.html.md.erb:<%= include "api-blueprint/Tutorial.md" %>
```

- https://apiblueprint.org/documentation/tutorial.html
- https://apiblueprint.org/documentation/specification.html
- https://apiblueprint.org/documentation/mson/specification.html